doc/classes/Object.xml

```diff
@@ -995,6 +996,7 @@
 				GD.Print(node.GlobalScale); // Prints Vector2(8, 2.5)
 				[/csharp]
 				[/codeblocks]
+				[b]Note:[/b] In GDScript, you can use index or attribute access instead. For example, [code]object["property"] = "value"[/code] or [code]object.property = "value"[/code]. Unlike [method set], these options produce an error if the property does not exist or the value is of an incompatible type.
```

Comment:
Let's remember who is reading these.
Also maybe sticking to the above example instead of being too generic?
```suggestion
				[b]Note:[/b] In GDScript, you can instead access properties directly ([code]node.global_scale = Vector2(8, 2.5)[/code]) or by index ([code]node["global_scale"] = Vector2(8, 2.5)[/code]). Unlike [method set], these options generate an error if the property does not exist or the value doesn't match.
```